src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -6682,8 +6686,28 @@ namespace SimAirServingZones {
 			FinalZoneSizing( CtrlZoneNum ).ZpzClgByZone = ZoneOAFrac;
 			// save new (increased) minimum flow rate
 			FinalZoneSizing( CtrlZoneNum ).DesCoolVolFlowMin = AvailSAFlow;
+			// make sure the max flow rate is >= the new minimum flow rate
+			FinalZoneSizing( CtrlZoneNum ).DesCoolVolFlow = max( AvailSAFlow, FinalZoneSizing( CtrlZoneNum ).DesCoolVolFlow );
```

Comment:
OK, so you calculate a new value of `DesCoolVolFlow` here.  Then it is used a few lines below in calculating a value for the local variables `VpzClgByZone` and `VdzClgByZone`.  And those are reported.  Is `DesCoolVolFlow` supposed to be used anywhere else?  I don't see it in the other places below.  But I'll keep looking.